CONTRIBUTING.md

```diff
@@ -68,3 +104,29 @@ Don't forget to add a short description of your change in the [CHANGELOG](CHANGE
  - Create github release https://github.com/plamere/spotipy/releases with the changelog content
    for the version and a short name that describes the main addition
  - Verify doc uses latest https://readthedocs.org/projects/spotipy/
+
+### Authors and Contributors
+
+Spotipy authored by Paul Lamere (plamere) with contributions by:
+
+Daniel Beaudry (danbeaudry on Github)
+Faruk Emre Sahin (fsahin on Github)
+George (rogueleaderr on Github)
+Henry Greville (sethaurus on Github)
+Hugo van Kemanade (hugovk on Github)
+José Manuel Pérez (JMPerez on Github)
+Lucas Nunno (lnunno on Github)
+Lynn Root (econchick on Github)
+Matt Dennewitz (mattdennewitz on Github)
+Matthew Duck (mattduck on Github)
+Michael Thelin (thelinmichael on Github)
+Ryan Choi (ryankicks on Github)
+Simon Metson (drsm79 on Github)
+Steve Winton (swinton on Github)
+Tim Balzer (timbalzer on Github)
+corycorycory on Github
+Nathan Coleman (nathancoleman on Github)
+Michael Birtwell (mbirtwell on Github)
+Harrison Hayes (Harrison97 on Github)
+Stephane Bruckert (stephanebruckert on Github)
+Ritiek Malhotra (ritiek on Github)
```

Comment:
As much as I appreciate every contributor, I don't think we need to plaster their names all over the place.

Reply:
True! We have https://github.com/spotipy-dev/spotipy/graphs/contributors for that

Also this list I outdated, I guess you've taken it from https://github.com/spotipy-dev/spotipy/blob/master/docs/index.rst#contribute - any chance you can remove that list from there too? We can just keep the technical instructions